core/win32/inject.c

```diff
@@ -1083,20 +1082,25 @@ generate_switch_mode_jmp_to_hook(HANDLE phandle, byte *local_code_buf,
     instr_t *restore_esp =
         INSTR_CREATE_mov_ld(GDC, opnd_create_reg(REG_ESP),
                             OPND_CREATE_MEM32(REG_NULL, (int)(size_t)mode_switch_data));
-
+	byte* eaxpos =(byte*) ((size_t)mode_switch_data) + 8;
```

Comment:
This does not satisfy the style conventions.  Best to use clang-format locally.  The clang-format GA CI run will fail unless this matches our style.

Reply:
Why +8?  Maybe should have a named offset or even better define a struct.

Reply:
+4 is fine. Defining a struct might be a bit overkill. I made variable in last commit.